site/src/docs/components/login/index.mdx

```diff
@@ -0,0 +1,150 @@
+---
+slug: '/components/login'
+title: 'Login'
+navTitle: 'Login'
+---
+
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+import ExternalLink from '../../../components/ExternalLink';
+import AnchorLink from '../../../components/AnchorLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+export const UsagePageAnchorLink = ({ anchor, children }) => {
+  return (
+    <AnchorLink path="/components/login/usage" anchor={anchor}>
+      {children}
+    </AnchorLink>
+  );
+};
+
+export const ApiPageAnchorLink = ({ anchor, children }) => {
+  return (
+    <AnchorLink path="/components/login/api" anchor={anchor}>
+      {children}
+    </AnchorLink>
+  );
+};
+
+## Introduction
+
+HDS Login components include React context, components and hooks for handling user authorisation, API tokens and session polling. React is not a requirement, but currently, no plain JavaScript UI components exist.
+
+Implementation requires an OIDC provider, which is a server for user authorization. The City of Helsinki uses <ExternalLink href="https://github.com/City-of-Helsinki/tunnistamo">Tunnistamo</ExternalLink>, but HDS login components can be used with any compatible OIDC provider. Read more about <ExternalLink href="https://auth0.com/intro-to-iam/what-is-openid-connect-oidc">OpenID Connect (OIDC)</ExternalLink>.
+
+In the OIDC login process, the browser window is redirected to the OIDC provider. Authentication is fully handled by the server. That is why HDS does not provide login or registration forms.
+
+When the user has been authenticated, the OIDC server redirects back to the service. HDS provides components to handle the authentication process initiation and responses.
+
+### Requirements
+
+Your service must be registered with an OIDC provider, like Tunnistamo.
+
+To use the OIDC provider with login components, you need
+
+- Client id.
+- Authority.
+- URL of the OIDC provider.
+
+### Important
+
+HDS <UsagePageAnchorLink>Oidc client</UsagePageAnchorLink> stores data in the session storage for security reasons. Session storage is not shared across browser windows or tabs, so the user must log in separately for each window.
```

Comment:
Because login system uses browser's session storage, maybe this Important section is a good place to say it clearly that this family of components doesn't work in SSR?

Reply:
Good idea, added a line for SSR.